tgui/packages/tgui/interfaces/Holopad.js

```diff
@@ -0,0 +1,157 @@
+import { Fragment } from 'inferno';
+import { useBackend } from '../backend';
+import { Box, Button, LabeledList, NoticeBox, Section } from '../components';
+import { Window } from '../layouts';
+
+export const Holopad = (props, context) => {
+  const { act, data } = useBackend(context);
+  const {
+    calling,
+  } = data;
+  return (
+    <Window resizable>
+      {calling && (
+        <Fragment>
+          <Box
+            bold
+            fontSize="36px"
+            mt={5}
+            mb={2}
+            textAlign="center"
+            fontFamily="monospace">
+            Dialing...
+          </Box>
+          <Box
+            textAlign="center"
+            fontSize="24px">
+            <Button
+              lineHeight="40px"
+              icon="times"
+              content="Hang Up"
+              color="bad"
+              onClick={() => act('hang_up')} />
+          </Box>
+        </Fragment>
```

Comment:
Use a Dimmer here perhaps, it also auto-centers everything.

Reply:
Used a Modal instead.